### PR TITLE
Add build script for spring-boot v3.0.1

### PR DESCRIPTION
**Pull request by Chandranana**
## Checklist

- [x] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [x] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [ ] Did you have **Legal approvals** for patch files ?

### CODE REVIEW

File: s/spring-boot/springboot_ubi_8.5.sh [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+#!/bin/bash
```
Comment (kandarpamalipeddi) [inline s/spring-boot/springboot_ubi_8.5.sh:21]: This statement is not required.

File: s/spring-boot/springboot_ubi_8.5.sh [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+VERSION=v3.0.1
```
Comment (kandarpamalipeddi) [inline s/spring-boot/springboot_ubi_8.5.sh:22]: Please make this version as configurable.
`VERSION=${1:-v3.0.1}`

**Review by kandarpamalipeddi — CHANGES_REQUESTED** [review]

File: s/spring-boot/springboot_ubi_8.5.sh (hunk 2 above)
Comment (Chandranana) [reply s/spring-boot/springboot_ubi_8.5.sh]: Done

File: s/spring-boot/springboot_ubi_8.5.sh (hunk 1 above)
Comment (Chandranana) [reply s/spring-boot/springboot_ubi_8.5.sh]: Done

**Review by kandarpamalipeddi — APPROVED** [review]

File: s/spring-boot/springboot_ubi_8.5.sh [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+yum install java-17-openjdk-devel.ppc64le -y
+
+#export variables
+export JAVA_HOME=/usr/lib/jvm/java-17-openjdk-17.0.5.0.8-2.el8_6.ppc64le
+
+#Build Spring boot
+git clone https://github.com/spring-projects/spring-boot
+cd spring-boot
+git checkout $VERSION
+./gradlew build
```
Comment (nikhil-kalbande) [inline s/spring-boot/springboot_ubi_8.5.sh:35]: @Chandranana - Aren't there any tests to run for this package. Any reason we are not running test here.

Comment (Chandranana) [reply s/spring-boot/springboot_ubi_8.5.sh:35]: gradlew build runs build and test

Comment (nikhil-kalbande) [reply s/spring-boot/springboot_ubi_8.5.sh:35]: I saw test command also so I was not sure. Thanks!

**Pull request merged**